go/vt/vtctl/workflow/server.go

```diff
@@ -3194,7 +3195,11 @@ func (s *Server) switchReads(ctx context.Context, req *vtctldatapb.WorkflowSwitc
 	if ts.MigrationType() == binlogdatapb.MigrationType_TABLES {
 		switch {
 		case ts.IsMultiTenantMigration():
-			ts.Logger().Infof("If keyspace routing rules are used, traffic is all or nothing per keyspace for workflow %s.%s", ts.targetKeyspace, ts.workflow)
+			err := sw.switchKeyspaceReads(ctx, roTabletTypes)
+			if err != nil {
+				return handleError(fmt.Sprintf("failed to switch read traffic for keyspace %s, workflow %s",
+					ts.SourceKeyspaceName(), ts.WorkflowName()), err)
```

Comment:
Do we want the source keyspace name here? I'm not sure if this is meant to say that we e.g. failed to switch traffic from source keyspace to target keyspace for workflow X or if it's meant more to be like targetkeyspace.workflowname? If it's the former then the message might be nicer if we explicitly say from X to Y.